docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -289,16 +361,48 @@ Step 1: Configure the Network on the First Node
 
         --peers tcp://172.0.1.3,tcp://172.0.1.4
 
-#. If the XO transaction processor is not running, start it as described in
-   Step 7 of :doc:`docker`.
+
+#. Start the REST API and transaction processors on the first validator node.
+   Open a separate terminal window to start each component.
+
+   .. note::
+
+      This network requires ``settings-tp`` and ``poet-validator-registry-tp``.
+      The other transaction processors (``intkey-tp-python`` and
+      ``xo-tp-python``) are not required, but are used for the other tutorials
+      in this guide. Note that each node in the network must run the same
+      transaction processors.
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth sawtooth-rest-api -v
```

Comment:
This command will use default validator component url. i.e. 127.0.0.1:4004
Could we add a note that the user would have to specify the validator component url, if its not the default one.

Refer: line 353:        --bind component:(component-bind-string} \